### ISSUE TITLE
quick question regarding batch norm for vgg models

### ISSUE THREAD

**Issue by rgeirhos** [brain-score/candidate_models#58]
Hi @mschrimpf,
I hope all is going well.
Quick question: for models "vgg-16" and "vgg-19" on brain-score.org, do these correspond to PyTorch models `torchvision.models.vgg16_bn` or to `torchvision.models.vgg16` (with / without batch norm)?
Cheers, Robert

**Comment by mschrimpf** [comment]
Hey @rgeirhos, great to read from you :)
VGGs in this repo actually use the keras implementations: https://github.com/brain-score/candidate_models/blob/745f9d1bc747e936cbdef1e7fc599b16cf9dc677/candidate_models/base_models/__init__.py#L362-L363 which do not use BatchNorm (https://github.com/keras-team/keras-applications/blob/master/keras_applications/vgg16.py). Note that e.g. keras/pytorch pretrained models tend to have slightly different weights and scores will differ accordingly

**Comment by rgeirhos** [comment]
Thanks Martin for your quick reply, this answers my question :)
